zavod/docs/best_practices/xpath_and_html.md

```diff
@@ -0,0 +1,30 @@
+# XPath and HTML
+
+## Selector Design
+Use semantic selectors that are resilient to HTML changes:
+
+```python
+# Good: 
+table = doc.xpath('.//div[@id="block-content"]//table')
+
+# Avoid:
+table = doc.xpath('.//div[@id="block-content"]//div[3]//table')
+```
+
```

Comment:
```suggestion
Prefer `div[contains(@class, 'abc')] over `div[@class='abc'] because classes can contain multiple values, but beware of conflicting class names like `abc-footer`
```